.bumpversion.cfg

```diff
@@ -24,3 +24,6 @@ serialize = {major}
 search = EVMC_ABI_VERSION = {current_version}
 replace = EVMC_ABI_VERSION = {new_version}
 
+[bumpversion:file:bindings/rust/evmc-sys/Cargo.toml]
```

Comment:
It should contain 
```
search = version = \"{current_version}\"
replace = version = \"{new_version}\"
```
or something...

Reply:
`bumpversion` seems to work without these